### ISSUE TITLE
Tier 1: Take Questions from DB, Put into Component

### ISSUE THREAD

**Issue by ogbanana** [meal-dot-match/Capstone#17]
- Grab the question from our DB
- Render it in the quiz component
- Render previous and next months (No prev for first-page, skip buttons, submit when the quiz is done)
- Pass results to results components from props.

Dependent on #16

**Comment by annkettles** [comment]
Updated Quiz component to pull in questions based on dummy data array. Waiting on seed file and db creation before refactoring. Will create another issue to remove dummy data and replace with real data.
